CHANGELOG.md

```diff
@@ -13,6 +28,10 @@ All significant changes to this repo will be summarized in this file.
 - Remove i18n/gettext task [\#331](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/331) ([DavidS](https://github.com/DavidS))
 - Restructure PuppetLint rake tasks so they can be configurable [\#330](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/330) ([nmaludy](https://github.com/nmaludy))
 
+**Merged pull requests:**
+
+- Release prep for v3.0.0 [\#333](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/333) ([DavidS](https://github.com/DavidS))
```

Comment:
Is this supposed to be here?

Reply:
it appears that all the release prep PRs go into this changelog. I've added the correct tags to the PR and it isn't ignored.

Reply:
Will go ahead and merge then